Libplanet.Stun/Stun/Messages/StunMessage.cs

```diff
@@ -170,17 +163,17 @@ public byte[] Encode(IStunContext ctx)
             using var ms = new MemoryStream();
             List<Attribute> attrs = Attributes.ToList();
 
-            if (!string.IsNullOrEmpty(ctx?.Username))
+            if (ctx.Username != string.Empty)
             {
                 attrs.Add(new Username(ctx.Username));
             }
 
-            if (ctx?.Nonce != null)
+            if (ctx.Nonce.Length > 0)
```

Comment:
`ctx?.Nonce != null` and `ctx.Nonce.Length > 0` is a different logic if `Nonce`'s length is 0. We need to check this change is ok for NineChronicles.

Reply:
Because the nonce type is byte[], the nonce property cannot be null.